ruby_basics/11_nested_collections/exercises/nested_hash_exercises.rb

```diff
@@ -31,7 +31,7 @@ def delete_language(languages, language_name)
 
 def find_beautiful_languages(languages)
   # Take languages and return a hash containing only languages which have the
-  # key/value pair { is_beautiful? => true } listed in their information
+  # key/value pair { :is_beautiful? => true } listed in their information
```

Comment:
```suggestion
  # key/value pair { is_beautiful?: true } listed in their information
```
In looking at the rest of this file, I think it would be better to update this hash to use the symbols syntax. This one was the only one using the rocket syntax.

Reply:
Agreed!